examples/qunit_lazy/tests.js

```diff
@@ -5,3 +5,12 @@ QUnit.test('says hello world', function(assert){
 QUnit.test('says hello to person', function(assert){
     assert.equal(hello('Bob'), 'hello Bob', 'should equal hello Bob');
 });
+
+QUnit.todo('failing todo', function(assert){
+    assert.ok(false, 'should be true');
+});
+
+// This will cause integration tests to fail
+// QUnit.todo('passing todo', function(assert){
+//     assert.ok(true, 'should be true');
+// });
```

Comment:
Do we still need this?

Reply:
I can take it out. I had it in there originally because I wanted to test that that case behaved correctly, but the integration test runner expects the tests to all pass. I didn't see a mechanism for intentional failure.

Reply:
Cleaning this up would be great :)

Reply:
@step2yeung I've updated the comment.